Podcast/DiscoverViewController.swift

```diff
@@ -124,91 +78,19 @@ class DiscoverViewController: UIViewController, UITableViewDelegate, UITableView
     
     func tableView(_ tableView: UITableView,
                             cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: "DiscoverTableViewCellIdentifier") as! DiscoverTableViewCell
-        cell.episode = feedEpisodes[indexPath.row]
-        cell.layoutSubviews()
+        let cell = tableView.dequeueReusableCell(withIdentifier: "FeedTableViewCellIdentifier") as! FeedTableViewCell
+        setupCellWithEpisode(cell: cell, card: feedEpisodes[indexPath.row])
         return cell
     }
     
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         
-        guard let cell = feedTableView.cellForRow(at: indexPath) as? DiscoverTableViewCell else { return }
-
-        cell.isExpanded = !cell.isExpanded
-
-        tableView.beginUpdates()
-        tableView.reloadData()
-        tableView.endUpdates()
+        guard let cell = feedTableView.cellForRow(at: indexPath) as? FeedTableViewCell else { return }
         
     }
     
-    //MARK: -
-    //MARK: CollectionView DataSource
-    //MARK: -
-    
-
-    func numberOfSections(in collectionView: UICollectionView) -> Int {
-        return 1
-    }
-    
-    func collectionView(_ collectionView: UICollectionView,
-                                 numberOfItemsInSection section: Int) -> Int {
-        return categories.count
-    }
-    
-    func collectionView(_ collectionView: UICollectionView,
-                                 cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "CategoriesCollectionViewIdentifier",
-                                                      for: indexPath) as! CategoriesCollectionViewCell
-        cell.categoryName = categories[indexPath.item]
-        cell.layer.cornerRadius = 2.0
-        cell.layer.masksToBounds = true
-        return cell
-    }
-    
-    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        print("selected \(categories[indexPath.item])")
-        let vc = CategoriesFeedViewController()
-        vc.category = categories[indexPath.item]
-        navigationController!.pushViewController(vc, animated: false)
-    }
-    
-    
-    //MARK: -
-    //MARK: Top Buttons
-    //MARK: -
-    
-    func categoriesButtonPress() {
-        trendingTagged = false
-        bottomLineView.frame.origin.x = view.frame.width / 2
-        feedTableView.removeFromSuperview()
-        view.addSubview(categoryCollectionView)
-        categoryCollectionView.reloadData()
-    }
-    
-    func trendingButtonPress() {
-        trendingTagged = true
-        bottomLineView.frame.origin.x = 0
-        categoryCollectionView.removeFromSuperview()
-        view.addSubview(feedTableView)
-        feedTableView.reloadData()
+    func setupCellWithEpisode(cell: FeedTableViewCell, card: Card) {
```

Comment:
This shouldn't have to be it's own function. Setting up cell with an episode should be a function on the cell class.